index.js

```diff
@@ -17,8 +17,24 @@ const EMAIL_PERMISSION = "alexa::profile:email:read";
 const EMPTY_STATE = 'empty';
 // used to move on to next step
 const TERMINATION_WORD = 'next';
+const helpWords = ['help'];
+const summaryWords = ['summary'];
+const restartWords = ['restart'];
 const MORE_INFO_PHRASE = 'Anything else?';
-const nextStepPhrases = ['nothing', 'nope', 'no', 'finish', 'next', 'continue', 'stop', 'that\'s it'];
+const nextStepPhrases = ['nothing', 'nope', 'no', 'finish', 'next', 'continue', 'stop', 'that\'s it', 'skip'];
+
+// TODO. make constants for questions.
+const QUESTION_PROJECT_NAME = "";
```

Comment:
fill in the constants with the hardcoded strings for each question at each step and replace them throughout the code